extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -384,6 +384,7 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
       'rc': this.fromResumeCallback ? 1 : null,
       'rafmt': this.getRafmtParam_(),
       'pfx': pfx ? '1' : '0',
+      'aanf': this.element.getAttribute('data-no-fill'),
```

Comment:
We should consider not making this start with "data-" given a pub could add it on their page and it would pass validation.   While a custom attribute would fail and that would be ok as you're adding during execution time.

Reply:
Done.